test/testenv/verificationutils.go

```diff
@@ -255,6 +258,42 @@ func VerifyServiceAccountConfiguredOnPod(deployment *Deployment, ns string, podN
 	}, deployment.GetTimeout(), PollInterval).Should(gomega.Equal(true))
 }
 
+// VerifyEphemeralStorage verifies volume 'mnt-splunk-etc' exists only after CR contains EphemeralStorage setting
```

Comment:
@jambrosiano As discussed offline, lets update the test case to modify a different spec parameter (such as CPU) to avoid slow tests